package.json

```diff
@@ -5,6 +5,7 @@
   "main": "./dist/react-scroll-to.js",
   "scripts": {
     "test": "jest",
+    "test.cover": "jest --coverage",
```

Comment:
Looks good! Can we change the script name to match the others? `test:cover`

Reply:
Updated 👍